docs/rest_api.md

```diff
@@ -416,6 +416,31 @@ Authorization requirements:
 Power cycle the node named `<node>`, and set it's next boot device to
 PXE. If the node is powered off, this turns it on.
 
+#### node_set_bootdev
+
+`POST /node/<node>/dev/<dev>/set_bootdev`
```

Comment:
Suggestion for a more rest-ish path: `/node/<node>/bootdev`. The general principle being that URLs are resources/objects, not procedures.